### DISCUSSION TITLE
use in ioBroker/Lovelace

### DISCUSSION THREAD

**Discussion by seba-78** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#521] [Q&A]
Hello,
I am using ioBroker/Lovelace and would love to use also this fantastic card. Unfortunately I cannot create an entity for devices in ioBroker, but only for individual states. As far as I understood I cannot configure this card, because you have to provide an entity for the whole vacuum device. 

All other lovelace cards have worked for me like a charm in ioBroker/Lovelace. 

thx for your help,
Sebastian

**Comment by PiotrMachowski** [comment]
Have you tried to use it nevertheless? With some luck it might work (at least partially) with some values of `vacuum_platform`. I'm not familiar with ioBroker, I have no idea how it works (let alone ioBroker.lovelace) though.

Which cards work on your setup?

It might be a better idea to ask this question in [ioBroker.lovelace](https://github.com/ioBroker/ioBroker.lovelace) repo, there should be more people that use it ;)

**Comment by seba-78** [comment]
thx for your quick answer! I use other cards such as battery-entity, button-card, mini-graph-card-bundle, etc. Nevertheless they do not need any devices as „entity folders“, but use directly state entities. Then it is no problem to configure these cards. 

I will go to the ioBroker lovelace repo or forum then. 🙏

**Reply by PiotrMachowski** [reply]
Vacuum map card is quite elastic, it can try to discover things by itself, but you can also configure it manually. Can you show your config for lovelace cards?
